output.txt

```diff
@@ -16,3 +16,9 @@ You are now connected to database "nc_news_test" as user "jamesyuill".
      0
 (13 rows)
 
+UPDATE 1
+ article_id |                title                | topic |    author     |         body          |     created_at      | votes |                                            article_img_url                                            
```

Comment:
This file shouldn't be in your PR (I assume this is you experimenting with some SQL)